solution/ui/regulations/eregs-component-lib/src/components/index.js

```diff
@@ -33,4 +32,8 @@ export { default as CopyCitation } from "./tooltips/CopyCitation.vue";
 export { default as GovInfoLinks } from "./tooltips/GovInfoLinks.vue";
 export { default as TooltipContainer } from "./tooltips/TooltipContainer.vue";
 
-// Other components you plan to publish go here
+// from Single Page App
+export { default as HeaderComponent } from "../../../eregs-vite/src/components/header/HeaderComponent.vue";
+export { default as HeaderLinks } from "../../../eregs-vite/src/components/header/HeaderLinks.vue";
+export { default as HeaderSearch } from "../../../eregs-vite/src/components/header/HeaderSearch.vue";
+export { default as JumpTo } from "../../../eregs-vite/src/components/JumpTo.vue";
```

Comment:
aliases in both `vite.config` files still need some ironing out to cut down on the need for relative paths like this